.github/workflows/ci.yml

```
@@ -101,6 +101,28 @@ jobs:
       - name: Run ESLint
         run: npm run lint
 
+  # Prettier check
+  format:
```

actionlint (1.7.10)

[error] "runs-on" section is missing in job "format"

(syntax-check)